packages/rebound-data/lib/rebound-data.js

```diff
@@ -5,19 +5,33 @@ import $ from "rebound-runtime/utils";
 
 
 var sharedMethods = {
+
```

Comment:
Will add a comment explaining what this does. When a change event propagates up the tree it modifies the path part of ```change:<path>``` to reflect the fully qualified path relative to that object. So:
```
model = { 
  obj: {
    arr: [ {val: 1}]
  }
}

model.set('obj.arr[0].val', 1);
```

Would trigger ```change:val```, ```change:[0].val```, ```change:arr[0].val``` and ```obj.arr[0].val``` on each parent as it is propagated up the tree.